flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/inference/InputTypeStrategies.java

```diff
@@ -172,8 +172,8 @@ public static InputTypeStrategy wildcardWithCount(ArgumentCount argumentCount) {
      * argument.
      */
     public static InputTypeStrategy comparable(
-            ConstantArgumentCount argumentCount, StructuredComparision requiredComparision) {
-        return new ComparableTypeStrategy(argumentCount, requiredComparision);
+            ConstantArgumentCount argumentCount, StructuredComparison requiredComparison) {
```

Comment:
I'm fine to get the hotfix in master branch, but not sure for 1.13, becuase it changes some public API (even we marked it `@Internal`). What do you think about this @twalthr ?

Reply:
Let's do it only in master. It is just a typo and not a critical bug that needs to be fixed urgently.

Reply:
+1

Reply:
Thanks @twalthr and @wuchong for the comment, If we only fix in master, it will fail when users want to load json serialized execution plan which generated from 1.13,  that's why i want to backport this one
CC @godfreyhe

Reply:
We also changed the JSON plan for a different issue recently. We do not provide any guarantees for this yet.

Reply:
It makes sense to me, I close this one.